lib/galaxy/tool_util/deps/containers.py

```diff
@@ -236,7 +236,11 @@ def __resolvers_dict(self):
 
     def find_best_container_description(self, enabled_container_types, tool_info, **kwds):
         """Yield best container description of supplied types matching tool info."""
-        resolved_container_description = self.resolve(enabled_container_types, tool_info, **kwds)
+        try:
+            resolved_container_description = self.resolve(enabled_container_types, tool_info, **kwds)
+        except Exception:
+            log.exception("Could not get container description")
```

Comment:
Should we also add `tool_info` in the log message?

Reply:
That's a good idea. The exception did include the quay.io URL, but I guess we may be catching other exception as well.